src/main/kotlin/org/koil/admin/AdminService.kt

```diff
@@ -41,35 +47,46 @@ class AdminServiceImpl(
         if ((adminEmailFromEnv.isNotEmpty() && adminPasswordFromEnv.isNotEmpty())
             && accountRepository.findAccountByEmailAddressIgnoreCase(adminEmailFromEnv) == null
         ) {
-            createAdminFromEmail(adminEmailFromEnv, HashedPassword.encode(adminPasswordFromEnv))
+            createDefaultAdminOrganization(adminOrganizationName)
```

Comment:
I've added a require() check to an init function within the AdminServiceImpl rather than checking within the method call. 

@stepbeekio let me know what you think about that as a solution - I figured it should fail at bean creation due to invalid configuration state rather then when after it starts up on the application event fire.

Reply:
Okay that makes sense.